demisto_sdk/commands/run_unit_tests/unit_tests_runner.py

```diff
@@ -67,6 +67,7 @@ def fix_coverage_report_path(coverage_file: Path) -> bool:
                         not (CONTENT_PATH / file).exists()
                         or file.parent.name
                         not in file.name  # For example, in `QRadar_v3` directory we only care for `QRadar_v3.py`
+                        or file.name == 'conftest.py'
```

Comment:
after a second thought we don't need this condition

This condition below should contain the `conftest` case
``` file.parent.name not in file.name``